cocos/spine/skeleton-cache.ts

```diff
@@ -302,7 +302,10 @@ export class AnimationCache {
     }
 
     public destroy (): void {
-        spine.SkeletonSystem.destroySpineInstance(this._instance);
+        if (this._instance) {
+            this._instance.destroy();
+            this._instance = null as any;
```

Comment:
If it could be null, should we define its type with ` | null`?

Reply:
If null is added to the declaration, all the places where it is used need to be checked for null, and there will be a bit more changes to be made.

Reply:
Yes，but that‘s the correct way since it could be `null`.